WordPress/src/main/res/values/strings.xml

```diff
@@ -2866,6 +2866,24 @@
     <string name="fab_add_category_desc">Create Category</string>
 
     <string name="content_description_more">More</string>
+
```

Comment:
1) Question (❓): Will this refactor somehow restarts `GlotPress` forcing another round of translations for these already translated strings?
2) Suggestion (💡): I would have reordered the `(Old)` strings to match the order of the `(New)` strings, just so that those can be easily match with each other and be present as they are being used in UI and code that binds everything together. Wdyt?

Reply:
1. I think as long as keys are the same, reorder should not matter. Otherwise, every time we add a new string, line number of existing strings change and they should get translated. Let me know if think otherwise. 
2. I ordered new strings based on the order they appear on the screen (working with you, I knew it would be discussed 😄), however, I missed applying the same order to the old strings thinking they'd anyways be removed. I've applied the same order now in 3d3b1d4.

Reply:
1. This indeed makes sense! 🤞 
2. ❤️ and on `(working with you, I knew it would be discussed 😄)` I am laughing out loud here... 🚀